Simbody/include/simbody/internal/Force_Custom.h

```diff
@@ -247,6 +247,15 @@ class SimTK_SIMBODY_EXPORT Force::Custom::Implementation {
     virtual bool dependsOnlyOnPositions() const {
         return false;
     }
+    /**
+     * Called during the realizeSubsystemDynamicsImpl method in
+     * GeneralForceSubsystemRep, used to determine whether a custom force should
```

Comment:
This comment is too detailed, and probably wrong now too. In any case the details of when it gets invoked are irrelevant to the API user. It should just explain under what circumstances the user should override the method, and what information that is conveying. Who uses it and how is not their business and we should be free to change that in the future without affecting the API.